packages/duplicate-package-checker-webpack-plugin/README.md

```diff
@@ -75,6 +75,10 @@ Packages with different major versions introduce backward incompatible changes a
 
 It is suggested that strict mode is kept enabled since this improves visibility into your bundle and can help in solving and identifying potential issues.
 
+## Node version support
+
+This package was developed and tested using Node 10 up to Node 14. This package has not been tested by us using Node 16 or greater. We don't want to restrict consumers on Node 16 or greater from using this package. For this reason the node version in package.json is set to `>=10` to allow Node 16 and above to use the package. No known issues were reported in newer Node versions but it is recommended that consumers to fully test it if you're using Node 16 or greater and report any issue.
```

Comment:
This reads a little weird to me.

```suggestion
This package was developed and tested using Node 10 up to Node 14. This package has not been thoroughly tested using Node 16 or greater. No known issues have been reported in newer Node versions, so this package is set to allow Node version 10 or greater. However, consumers utilizing Node 16 or greater should use this package at their own risk. 
```

Reply:
Updated for all packages: https://github.com/cerner/terra-toolkit/pull/731/commits/b6204b164bf0d7c8e055afabc376fb83a3763708